CONTRIBUTING.md

```diff
@@ -64,18 +64,21 @@ From the top-level xrpl.js folder (one level above `packages`), run the followin
 ```bash
 npm install
 # sets up the rippled standalone Docker container - you can skip this step if you already have it set up
-docker run -p 6006:6006 --interactive -t --volume $PWD/.ci-config:/opt/ripple/etc/ --platform linux/amd64 rippleci/rippled:2.0.0-b4 /opt/ripple/bin/rippled -a --conf /opt/ripple/etc/rippled.cfg
+docker run  -p 6006:6006 --rm -it --name rippled_standalone --volume $PWD/.ci-config:/etc/opt/ripple/ --entrypoint bash rippleci/rippled:2.3.0-rc1 -c 'rippled -a'
+0-b4 /opt/ripple/bin/rippled -a --conf /opt/ripple/etc/rippled.cfg
```

Comment:
what is `0-b4` in this line? 
If you look into the xrpl-py implementation of Verifiable Credentials, there is a way to obviate specifying the configuration file path. I feel that is an easier solution for developers.

Reply:
That whole line (68) was old, part of a copy paste issue. Have deleted it, the actual command was just line 67